src/ILVerify/src/Program.cs

```diff
@@ -97,6 +94,9 @@ private ArgumentSyntax ParseCommandLine(string[] args)
             foreach (var reference in referenceFiles)
                 Helpers.AppendExpandedPaths(_referenceFilePaths, reference, false);
 
+            if (string.IsNullOrEmpty(_systemModule))
```

Comment:
The pattern we have been using for defaults is to set them before the parsing the command line options, like: https://github.com/dotnet/corert/blob/master/src/ILCompiler/src/Program.cs#L38